apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -42,6 +42,8 @@
  ****************************************************************************/
 #define AUDIO_TEST_FILE "/mnt/pcm"
 
+#define AUDIO_TEST_FILE1 "/mnt/pcm1"
```

Comment:
FILE1 & FILE2 is better

Reply:
Actually FILE1 is not needed, I have removed that definition.